django/docker-compose.django.yml

```diff
@@ -110,21 +69,30 @@ services:
             GODEBUG: "netdns=go"
         # We use internal networks:
         # - 'server-postgresql' - for AcraServer and DB interconnection
-        # - 'connector-server' - for AcraServer and AcraConnector interconnection
+        # - 'django-server' - for AcraServer and Django interconnection
         networks:
-            - connector-server
+            - django-server
             - server-postgresql
         volumes:
             # Mount the directory with only the keys for this service. Must be
             # rewriteable in case of using API, otherwise should be read-only.
             - ./.acrakeys/acra-server:/keys
             # Directory with configuration, rewriteable
             - ./.acraconfigs/acra-server:/config
+            - ../_common/ssl/acra-server/:/ssl
+            - ../_common/ssl/root.crt:/ssl/root.crt
         command: >-
             --db_host=postgresql
             --db_port=5432
             --keys_dir=/keys
             --http_api_enable
+            --tls_auth=4
+            --tls_ca=/ssl/root.crt
+            --tls_cert=/ssl/acra-server.crt
+            --tls_key=/ssl/acra-server.key
+            --tls_client_id_from_cert
+            --tls_identifier_extractor_type=distinguished_name
+            --acraconnector_transport_encryption_disable
```

Comment:
this flag should be here, right?

Reply:
yes, it turns off acra-connector usage and waiting of trace/clientID info from connector side, and expects raw connection. in this case it is TLS connection